app/calendar/outlookcalendar/outlookcalendar.ts

```diff
@@ -0,0 +1,120 @@
+import { CalendarService, CreateEventBooking } from "app/calendar/calendar-service"
+import {
+  AuthorizationHeader,
+  getAuthorizationHeader,
+} from "app/calendar/outlookcalendar/helper/getAuthorizationHeader"
+import { addSeconds } from "date-fns"
+import { ConnectedCalendar } from "db"
+import { boilDownTimeIntervals } from "app/calendar/utils/boildown-intervals"
+import makeRequestTo from "app/calendar/outlookcalendar/helper/callMicrosoftAPI"
+
+export class OutlookCalendarService implements CalendarService {
+  private authorizationHeader: AuthorizationHeader
+  private calendar: ConnectedCalendar
+
+  constructor(calendar: ConnectedCalendar) {
+    if (!calendar.refreshToken) {
+      throw new Error("refreshToken missing!")
+    }
+    this.calendar = calendar
+  }
+
+  async initialize() {
+    this.authorizationHeader = await getAuthorizationHeader(this.calendar.refreshToken!)
+  }
+
+  public async createEvent(booking: CreateEventBooking) {
+    const url = new URL("https://graph.microsoft.com/v1.0/me/calendar/events")
+    const startDate = booking.startDateUTC
+    const endDate = addSeconds(booking.startDateUTC, booking.meeting.duration)
+    const body = {
+      Subject: booking.meeting.name + " with " + booking.inviteeEmail,
+      Body: {
+        ContentType: "HTML",
+        Content: "This meeting was booked via kalle.app",
+      },
+      Start: {
+        DateTime: startDate,
+        timeZone: "UTC",
+      },
+      End: {
+        DateTime: endDate,
+        timeZone: "UTC",
+      },
+      Attendees: [
+        {
+          EmailAddress: {
+            Address: booking.inviteeEmail,
+            Name: booking.inviteeEmail.split("@")[0],
+          },
+          Type: "Required",
+        },
+      ],
+    }
+
+    var options = {
```

Comment:
```suggestion
    const options = {
```

(or even in-line it into the `makeRequestTo` call)